libkpod/attach.go

```diff
@@ -0,0 +1,182 @@
+package libkpod
```

Comment:
Why is this in libkpod rather then libpod, I thought we were done with libkpod?

Reply:
ah because I have been working on this for quite some time ?